project-api/project-api-test/src/test/resources/balawriter/projectOne/Compiler-plugin.toml

```diff
@@ -0,0 +1,5 @@
+[plugin]
+class = "io.ballerina.openapi.Validator"
+
+[[Dependency]]
```

Comment:
This should be `dependencies` right since this is a collection ? can you cross check with Dependencies.toml

Reply:
Made this `Dependency` after this comment
https://docs.google.com/document/d/1e0wu1LiDknj7RA43H4Qtu3FWZPzKSMyrBpikXgt9GHE/edit?disco=AAAAH-gUJ1k